src/modules/ekf2/EKF/RingBuffer.h

```diff
@@ -37,6 +37,9 @@
  * Template RingBuffer.
  */
 
+#ifndef EKF_RINGBUFFER_H
```

Comment:
why not using `#pragma once` ?

Reply:
It's because of the optimized "unity build" where all source is combined into a single compilation unit which allows the compiler to inline across boundaries. 

https://github.com/PX4/PX4-Autopilot/blob/82f63475d7f97056171fff1a79eddfa266441041/src/modules/ekf2/CMakeLists.txt#L129